tools/mo/unit_tests/mo/middle/StridedSliceNormalizer_test.py

```diff
@@ -1644,96 +1650,44 @@ def test_strided_slice_permute_2(
             shrink_axis_mask=(0, 1, 0, 0), new_axis_mask=(0, 0, 0, 0), ellipsis_mask=(0, 0, 0, 0)
     ):
         self.run_permute_test(inp, ref_res, begin, end, strides,
                               begin_mask, end_mask, shrink_axis_mask, new_axis_mask, ellipsis_mask)
 
     def test_strided_slice_permute_4(
-            self,  # inp[1:34, :, :, :2] begin mask is (1,) so only one value can be specified
-            inp=(1, 35, 35, 3), ref_res=(1, 2, 35),
-            begin=(0, 0, 0, 0), end=(1, 34, 20, 2), strides=(1, 1, 1, 1), begin_mask=(1, 0, 0, ), end_mask=(1, 0, 0, 1),
-            shrink_axis_mask=(0, 1), new_axis_mask=(0,), ellipsis_mask=(0,)
+            # no shrink/new axis therefore will be permuted
+            # inp[0:1, :, :, :2] begin mask is (1,) so only one begin value need to be specified
+            self,
+            inp=(16, 35, 35, 3), ref_res=(1, 2, 35, 35),
+            begin=(0, 0, 0, 0), end=(1, 34, 20, 2), strides=(1, 1, 1, 1), begin_mask=(1, 0, 0,), end_mask=(1, 0, 0, 1),
+            shrink_axis_mask=(0,), new_axis_mask=(0,), ellipsis_mask=(0,)
     ):
         self.run_permute_test(inp, ref_res, begin, end, strides,
                               begin_mask, end_mask, shrink_axis_mask, new_axis_mask, ellipsis_mask)
 
     def test_strided_slice_permute_5(
             self,  # inp[:, :, :, :] since all begin and end masks are zero
             inp=(1, 35, 35, 3), ref_res=(1, 3, 35, 35),
-            begin=(1, 10, 10, 0), end=(1, 34, 20, 2), strides=(1, 1, 1, 1), begin_mask=(0, 0, 0, 0), end_mask=(0, 0, 0, 0),
+            begin=(1, 10, 10, 0), end=(1, 34, 20, 2), strides=(1, 1, 1, 1), begin_mask=(0, 0, 0, 0),
+            end_mask=(0, 0, 0, 0),
             shrink_axis_mask=(0,), new_axis_mask=(0,), ellipsis_mask=(0,)
     ):
         self.run_permute_test(inp, ref_res, begin, end, strides,
                               begin_mask, end_mask, shrink_axis_mask, new_axis_mask, ellipsis_mask)
 
     def test_strided_slice_permute_6(
             self,  # inp[0]
-            inp=(1, 35, 35, 3), ref_res=(3, 35, 35),
+            inp=(1, 35, 35, 3), ref_res=(35, 35, 3),
             begin=(0,), end=(1,), strides=(1,), begin_mask=(1,), end_mask=(0,),
             shrink_axis_mask=(1,), new_axis_mask=(0,), ellipsis_mask=(0,)
     ):
         self.run_permute_test(inp, ref_res, begin, end, strides,
                               begin_mask, end_mask, shrink_axis_mask, new_axis_mask, ellipsis_mask)
 
     def test_strided_slice_permute_7(
-            self,  # inp[0, 20], ends can be of any value
-            inp=(1, 35, 35, 3), ref_res=(3, 35),
-            begin=(0, 20), end=(1, 9999), strides=(1, 1), begin_mask=(0,), end_mask=(0,),
-            shrink_axis_mask=(1, 1), new_axis_mask=(0,), ellipsis_mask=(0,)
-    ):
-        self.run_permute_test(inp, ref_res, begin, end, strides,
-                              begin_mask, end_mask, shrink_axis_mask, new_axis_mask, ellipsis_mask)
-
-    def test_strided_slice_permute_8(
```

Comment:
why did you remove a lot of tests? Please recover them if possible

Reply:
I removed them, because at the beginning I added them to check that attributes are changed accordingly with layout but now since for some cases (when srink/new axis are nonzero) attributes are not changed when with layout. 

So I removed them because they were red (expectedly) and corrected some others.